packages/sit-onyx/src/i18n/locales/de-DE.json

```
@@ -17,6 +17,9 @@
       "url": "\"{value}\" muss eine gültige URL sein."
     }
   },
+  "select": {
+    "toggleDropDown": "Toggle Dropdown"
```

```suggestion
    "toggleDropDown": "Auswahl umschalten"
```

Not sure whats the German translation for "Toggle select"^^